types/genesis.go

```diff
@@ -0,0 +1,10 @@
+package types
```

Comment:
it isn't used anywhere, can we delete it?

Reply:
oops dont know it got here, surpose to be on the follow up PR